### PR TITLE
Adding type index and disabling warns and debug statements

### PR DESCRIPTION
**Pull request by malware-unicorn**
Adding type_index to procedure symbol and disabling warns and debug statements.

### CODE REVIEW

File: crates/ezpdb/Cargo.toml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-[dependencies]
-pdb = "0.7"
-thiserror = "1.0"
-log = "0.4"
-serde = { version = "1.0", features = ['derive', 'rc'], optional = true }
-uuid = "0.8"
-
-[patch.crates-io]
-pdb = { git = "https://github.com/landaire/pdb"}
+[package]
```
Comment (landaire) [inline crates/ezpdb/Cargo.toml]: This looks like just a whitespace change? `LF` -> `CRLF`? Could you remove this from the PR if so?

File: crates/ezpdb/src/lib.rs [hunk 2]
```diff
@@ -22,7 +22,7 @@ pub fn parse_pdb<P: AsRef<Path>>(
     base_address: Option<usize>,
 ) -> Result<ParsedPdb, crate::error::Error> {
     let file = File::open(path.as_ref())?;
-    debug!("opening PDB");
+    //debug!("opening PDB");
```
Comment (landaire) [inline crates/ezpdb/src/lib.rs]: I kind of did a side channel here to figure out why this was needed. The debug logic is necessary for debugging my own issues in another tool. If you don't want it, you should configure the logger you're using to filter out this module/crate: https://docs.rs/env_logger/latest/env_logger/struct.Builder.html#method.filter_module

**Review by landaire — CHANGES_REQUESTED** [review]

File: crates/ezpdb/Cargo.toml (hunk 1 above)
Comment (malware-unicorn) [reply crates/ezpdb/Cargo.toml]: Thanks, reverted this.

**Comment by landaire** [comment]
@malware-unicorn thanks!

File: crates/ezpdb/src/lib.rs (hunk 2 above)
Comment (malware-unicorn) [reply crates/ezpdb/src/lib.rs]: Reverted the changes here, will use the env_logger fix.

**Pull request merged**